src/Symfony/Component/Yaml/Parser.php

```diff
@@ -336,7 +336,8 @@ private function getNextEmbedBlock($indentation = null, $inSequence = false)
                 $removeComments = !preg_match($removeCommentsPattern, $this->currentLine);
             }
 
-            if ($isItUnindentedCollection && !$this->isStringUnIndentedCollectionItem($this->currentLine)) {
+            $isStringUnIndentedShortcutKey = (2 === ($indent - $newIndent));
```

Comment:
@aitboudad Honestly, I don't see a (semantical) difference between your two examples.

Could we remeber the number of spaces between the dash and the list item when detecting such constructs to not use a hard-coded value of `2` here?